src/Factory/ApiProblemListenerFactory.php

```diff
@@ -4,8 +4,8 @@
 
 namespace Laminas\ApiTools\ApiProblem\Factory;
 
-use Interop\Container\ContainerInterface;
 use Laminas\ApiTools\ApiProblem\Listener\ApiProblemListener;
+use Psr\Container\ContainerInterface;
```

Comment:
This change is a potential breaking change for downstream extending classes: to avoid that, we probably need to include `laminas/laminas-servicemanager` as a polyfill :thinking:

Reply:
@Ocramius 
You are right, the factories are not final. :tired_face:

Reply:
I hate how it took us a decade to realize how problematic non-`final` code is :(

Reply:
I added `laminas/laminas-servicemanager` to the `composer.json` file. Let me know if I should modify anything else.